### PR TITLE
adding no follow

### PR DESCRIPTION
**Pull request by Daniellem97**
# Description of the change

## Checklist

Please make sure that the proposed change checks all the boxes below before requesting a review:

- [x] I have reviewed the [guidelines for contributing](https://github.com/spacelift-io/user-documentation/blob/main/CONTRIBUTING.md) to this repository.
- [x] The preview looks fine.
- [x] The tests pass.
- [x] The commit history is clean and meaningful.
- [x] The pull request is opened against the `main` branch.
- [x] The pull request is no longer marked as a draft.
- [x] You agree to license your contribution under the [MIT license](https://github.com/spacelift-io/user-documentation/blob/main/LICENSE) to Spacelift (not required for Spacelift employees).
- You have updated the navigation files correctly:
    - [ ] No new pages have been added, or;
    - [ ] Only _nav.yaml_ has been updated because the changes only apply to SaaS, or;
    - [ ] Only _nav.self-hosted.yaml_ has been updated because the changes only apply to Self-Hosted, or;
    - [ ] Both _nav.yaml_ and _nav.self-hosted.yaml_ have been updated.

If the proposed change is ready to be merged, please request a review from `@spacelift-io/solutions-engineering`. Someone will review and merge the pull request.

_Spacelift employees should request reviews from the relevant engineers and are allowed to merge pull requests after they got at least one approval._

Thank you for your contribution! 🙇

### CODE REVIEW

File: docs/concepts/spacectl.md [hunk 1]
```diff
@@ -83,23 +83,23 @@ steps:
 
 #### Arch linux
 
-Install [`spacectl-bin`](https://aur.archlinux.org/packages/spacectl-bin) from the Arch User Repository ([AUR](https://aur.archlinux.org/)):
+Install [`spacectl-bin`](https://aur.archlinux.org/packages/spacectl-bin): from the Arch User Repository ([AUR](https://aur.archlinux.org/)):
```
Comment (eminalemdar) [inline docs/concepts/spacectl.md:86]: missing the nofollow argument here

File: docs/integrations/cloud-providers/azure.md [hunk 2]
```diff
@@ -121,7 +121,7 @@ This will output something like the following:
 ```bash
 Changing "spacelift-sp" to a valid URI of "http://spacelift-sp", which is the required format used for service principal names
 Creating 'Contributor' role assignment under scope '/subscriptions/458fd769-5a4c-4df2-a339-8981094d8899'
-The output includes credentials that you must protect. Be sure that you do not include these credentials in your code or check the credentials into your source control. For more information, see [Create an Azure service principal with the Azure CLI](https://aka.ms/azadsp-cli){: rel="nofollow"}
+The output includes credentials that you must protect. Be sure that you do not include these credentials in your code or check the credentials into your source control. For more information, see [Create an Azure service principal with the Azure CLI](https://aka.ms/azadsp-cli)
```
Comment (eminalemdar) [inline docs/integrations/cloud-providers/azure.md:124]: why was this removed here?

**Review by eminalemdar — CHANGES_REQUESTED** [review]

File: docs/integrations/cloud-providers/azure.md (hunk 2 above)
Comment (Daniellem97) [reply docs/integrations/cloud-providers/azure.md]: removed as it is within a script

File: docs/concepts/spacectl.md [hunk 3]
```diff
@@ -83,23 +83,23 @@ steps:
 
 #### Arch linux
 
-Install [`spacectl-bin`](https://aur.archlinux.org/packages/spacectl-bin) from the Arch User Repository ([AUR](https://aur.archlinux.org/)):
+Install [`spacectl-bin`](https://aur.archlinux.org/packages/spacectl-bin): from the Arch User Repository ([AUR](https://aur.archlinux.org/){: rel="nofollow"}):
```
Comment (eminalemdar) [inline docs/concepts/spacectl.md:86]: I think we should also have the `rel="nofollow"` for the first link too

**Review by eminalemdar — CHANGES_REQUESTED** [review]

**Review by eminalemdar — APPROVED** [review]

**Pull request merged**